app/components/support/school_details_summary_list_component.rb

```diff
@@ -1,11 +1,33 @@
 class Support::SchoolDetailsSummaryListComponent < ResponsibleBody::SchoolDetailsSummaryListComponent
   def rows
-    super +
-      (headteacher.present? ? [headteacher_row] : [])
+    array = super
+    array << headteacher_row if headteacher.present?
+    array.insert(array.find_index { |row| row[:key] == 'Can place orders?' }, mno_row) if @school.mno_feature_flag?
```

Comment:
Part of me wonders whether we should extract all these row keys into one place  like a constant or a translation in `en.yml`.

Reply:
Yea, we have a few magic strings floating about